api/src/test/java/org/confeti/controllers/core/StatisticControllerTestUtils.java

```diff
@@ -0,0 +1,33 @@
+package org.confeti.controllers.core;
+
+import org.springframework.test.web.reactive.server.EntityExchangeResult;
+import org.springframework.test.web.reactive.server.WebTestClient;
+
+import java.util.List;
+
+import static org.confeti.controllers.ControllersTestsUtils.assertEqualsList;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+public class StatisticControllerTestUtils {
+
+    public static <T> void testGetListResponse(
```

Comment:
You can use `assertThat` and `containsInAnyOrder` from hamrest and remove `assertEqualsList` with `ObjectMapper`:
```
public static <T> void testGetListResponse(final Object controller,
                                               final String uri,
                                               final List<T> expectedResponse,
                                               final Class<T[]> clazz) {
        final var res = WebTestClient
                .bindToController(controller)
                .build()
                .get()
                .uri(uri)
                .exchange()
                .expectStatus().isOk()
                .expectBody(clazz).returnResult();
        assertThat(Arrays.asList(res.getResponseBody()), containsInAnyOrder(expectedResponse.toArray()));
    }
```